From b67390e48f0952424272aeb1191c80a595e4d73f Mon Sep 17 00:00:00 2001
From: Yicong Yang <yangyicong@hisilicon.com>
Date: Fri, 26 Nov 2021 16:26:59 +0800
Subject: [PATCH] openeuler_defconfig: Build HiSilicon I2C/SPI/SFC driver as
 module
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: b67390e48f0952424272aeb1191c80a595e4d73f


driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4J9B0?from=project-issue

------------------------------------------------------------------------

Build HiSilicon I2C/SPI/SFC driver as module.

Signed-off-by: Yicong Yang <yangyicong@hisilicon.com>
Reviewed-by: Jay Fang <f.fangjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 76d6a118330d..34aa2da5bf15 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -3378,6 +3378,7 @@ CONFIG_I2C_DESIGNWARE_PLATFORM=m
 # CONFIG_I2C_EMEV2 is not set
 CONFIG_I2C_GPIO=m
 # CONFIG_I2C_GPIO_FAULT_INJECTOR is not set
+CONFIG_I2C_HISI=m
 # CONFIG_I2C_NOMADIK is not set
 # CONFIG_I2C_OCORES is not set
 CONFIG_I2C_PCA_PLATFORM=m
@@ -3431,7 +3432,8 @@ CONFIG_SPI_DESIGNWARE=y
 # CONFIG_SPI_DW_DMA is not set
 CONFIG_SPI_DW_PCI=m
 CONFIG_SPI_DW_MMIO=y
-# CONFIG_SPI_HISI_SFC_V3XX is not set
+CONFIG_SPI_HISI_KUNPENG=m
+CONFIG_SPI_HISI_SFC_V3XX=m
 # CONFIG_SPI_NXP_FLEXSPI is not set
 # CONFIG_SPI_GPIO is not set
 # CONFIG_SPI_FSL_SPI is not set
-- 
2.26.2

